src/Codegen/Constraints/ArrayBuilder.php

```diff
@@ -31,7 +31,7 @@ public function build(): this {
     $class = $this->codegenClass()
       ->addMethod($this->getCheckMethod());
 
-    $properties = [];
+    $properties = vec[];
```

Comment:
`$hb->addMultilineCall` does not care about what type of Container you give it. It actually takes a `Traversable<string>`, but it passes it to implode, which requires a Container be given. I'll make a PR there to fix this error.